Apps/phresilient/readme.html

```diff
@@ -0,0 +1,5 @@
+<!-- File: readme.html
+  Copyright (c) 2020 Splunk Inc.
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+<html><head></head><body>Replace this text in the app's <b>readme.html</b> to contain more detailed information</body></html>
```

Comment:
No valid information in this readme. You might want to move readme.1st's text into here